### ISSUE TITLE
API ReadTheDocs failing

### ISSUE THREAD

**Issue by laurenmarietta** [spacetelescope/jwql#437]
It seems as though our webhook to ReadTheDocs is not currently updating the API docs:
[![Documentation Status](https://readthedocs.org/projects/jwql/badge/?version=latest)](https://jwql.readthedocs.io/en/latest/?badge=latest)

Actually, it looks like our doc builds have been failing for a month!
https://readthedocs.org/projects/jwql/builds/

There is an error message here that should be helpful for investigating: https://readthedocs.org/projects/jwql/builds/9177249/

**Comment by bourque** [comment]
@catherine-martlin I made just a few tweaks (#474) to get things working.  The main problem actually was that RTD was pointing to the `master` branch, and some of the fixes were simply just not merged into `master` yet (e.g. #463).  I went into the advanced RTD settings to switch the default branch to `develop` and builds started passing.

I think we should probably just keep pointing to `develop` branch anyways since that is what we mostly use right now.  Perhaps we can switch back to `master` after our 1.0 release and after we actually have a user base.
